notifications/urls.py

```
@@ -0,0 +1,33 @@
+from .views import NotificationViewSet, ClinicalAlertViewSet
+from .smart_reminder_views import (
+    SmartReminderViewSet,
+    ReminderPatternViewSet,
+    ReminderScheduleViewSet,
+    ReminderResponseViewSet,
+    SmartReminderServiceViewSet
+)
+
+router = DefaultRouter()
+
+# اطلاع‌رسانی‌ها و هشدارها
```

Ruff (0.12.2)

Comment contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)